src/lib/components/schedule/schedules-table-row.svelte

```diff
@@ -24,22 +24,23 @@
   <td class="cell">
     <WorkflowStatus status={schedule?.info?.paused ? 'Paused' : 'Running'} />
   </td>
-  <td class="cell truncate">
-    {schedule.scheduleId}
+  <td class="cell whitespace-pre-line break-words">
+    <p class="text-base">{schedule.scheduleId}</p>
     <p>
-      <small class="text-gray-900"
-        ><ScheduleFrequency
-          calendar={schedule?.info?.spec?.calendar?.[0]}
-          interval={schedule?.info?.spec?.interval?.[0]}
-        /></small
-      >
+      <ScheduleFrequency
+        calendar={schedule?.info?.spec?.structuredCalendar?.[0]}
+        interval={schedule?.info?.spec?.interval?.[0]}
+        class="text-sm"
+      />
     </p>
   </td>
-  <td class="cell hidden md:table-cell">
-    {schedule?.info?.workflowType?.name}
+  <td class="cell hidden whitespace-pre-line break-words md:table-cell">
+    {schedule?.info?.workflowType?.name ?? ''}
   </td>
-  <td class="cell links hidden xl:table-cell">
-    {#each schedule?.info?.recentActions?.reverse().slice(0, 5) ?? [] as run}
+  <td class="cell links hidden truncate xl:table-cell">
+    {#each schedule?.info?.recentActions
+      ?.sort((a, b) => new Date(b.actualTime).getTime() - new Date(a.actualTime).getTime())
+      .slice(0, 5) ?? [] as run}
```

Comment:
Might be worth extracting this into a function that can be tested, or at least a reactive property up in the `<script />`